test/Microsoft.Extensions.Options.Test/OptionsTest.cs

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
```

Comment:
Curious, is this file new or a rename/move? It seems that the diff is missing something.